import "hero.proto";

message PlayerResponse
{
    required int64 id = 1;
    required string nickname = 2;
    repeated Hero_PB hero_list = 3;//武将列表
    //todo 装备列表
    //todo 活动信息


}